packages/swapper/src/swappers/thorchain/utils/txFeeHelpers/ethTxFees/getEthTxFees.ts

```diff
@@ -41,35 +29,20 @@ export const getEthTxFees = async ({
       )
     }
 
-    let feeDataOptions
-    try {
-      feeDataOptions = await adapter.getFeeData({
```

Comment:
I removed this because it was failing almost every time. Removing this will speed up quotes